docs/src/integrations/daemon.md

```diff
@@ -103,14 +103,28 @@ Here are the available options and fields you can use in the builder object:
-- `fee_granter` (*optional*) allows you to use the fee-grant module. If this field is specified, the sender will try to pay for transactions using the specified `granter`. <a href="https://docs.cosmos.network/v0.46/modules/feegrant/" target="_blank">More info on the fee grant module</a>. <a href="https://docs.rs/cw-orch-daemon/latest/cw_orch_daemon/sync/struct.DaemonBuilder.html#method.fee_granter" target="_blank">Documentation Link</a>
-- `hd_index` (*optional*) allows to set the index of the HD path for the account associated with the `Daemon` object. <a href="https://hub.cosmos.network/main/resources/hd-wallets.html" target="_blank">More info on the derivation path and index</a>. <a href="https://docs.rs/cw-orch-daemon/latest/cw_orch_daemon/sync/struct.DaemonBuilder.html#method.hd_index" target="_blank">Documentation Link</a>
+- `state` (*optional*) is used when you want to specify an existing `DaemonState` object to the new Daemon. This is particularly useful when interacting with multiple chains at the same time.
 
-> **NOTE**: if none of `sender` or `mnemonic` is specified, [env variables](../contracts/env-variable.md) will be used to construct the sender object.
+> **NOTE**: if `mnemonic` is not specified, [env variables](../contracts/env-variable.md) will be used to construct the sender object.
 
-Keep in mind that those options can't be changed once the `Daemon` object is built, using the `build` function. It is possible to create a new `DaemonBuilder` structure from a `Daemon` object by using the `rebuild` method and specifying the options that you need to change.
+Keep in mind that most of these options can't be changed once the `Daemon` object is built, using the `build` function. It is possible to create a new `DaemonBuilder` structure from a `Daemon` object by using the `rebuild` method and specifying the options that you need to change.
+
+### Properties of the default sender
+
+If you wish to use the default `CosmosSender` provided by default, you can use this simple pattern:
+
+```rust,ignore
+  let daemon = Daemon::builder(JUNO_1).build()?;
+```
+
+You can use the following functions on the `Daemon` object to customize some of its properties : 
+
+- `Daemon::authz_granter` allows you to use the authz module. If this method is used, the sender will send transactions wrapped inside an authz message sent by the specified `granter`. <a href="https://docs.cosmos.network/v0.46/modules/authz/" target="_blank">More info on the authz module</a>. <a href="https://docs.rs/cw-orch-daemon/latest/cw_orch_daemon/type.Daemon.html#method.authz_granter" target="_blank">Documentation Link</a>
```

Comment:
Honestly I'm not sure if exposing these functions is that helpful VS confusing for users. If it's already exposed on the sender, should we expose it on the daemon as well?

@Kayanski @Buckram123

Reply:
It would look like this is the users code, that's bad for starting users but then again, this functionality is not necessarily for starters: 
```rust
daemon.sender_mut().set_authz_granter(granter);
```

Reply:
Yeah I do prefer that. 

Or they construct `CosmosSender` and configure it with authz before building with `build_sender(cosmos_sender)`

Reply:
Those two ways is already enough imo.